stellar/utils.ts

```diff
@@ -193,14 +204,13 @@ async function broadcast(operation, wallet, chain, action, options = {}, simulat
 
     if (simulateTransaction) {
         const tx = await buildTransaction(operation, server, wallet, chain.networkType, options);
-        const response = await server.simulateTransaction(tx);
-
-        if (response.error) {
-            throw new Error(response.error);
+        try {
+            const response = await server.simulateTransaction(tx);
+            printInfo('successfully simulated tx', `action: ${action}, networkType: ${chain.networkType}, chainName: ${chain.name}`);
```

Comment:
Let the caller print if they wish instead? Verbose logging from a helper by default creates noise